src/nmea.h

```diff
@@ -32,11 +32,23 @@
 #define NMEA_GGA_FIX_MANUAL  7
 #define NMEA_GGA_FIX_SIM     8
 
+#define RAD2DEG_INT(X) (s16)((180.0 / M_PI) * X)
+#define RAD2DEG_FLOAT(X) ((180.0/M_PI) * X)
```

Comment:
we also have these already defined:
https://github.com/swift-nav/libswiftnav/blob/master/include/libswiftnav/constants.h#L25